docs/getting_started.md

```diff
@@ -218,33 +214,29 @@ Back in the **Covariate Data** datasheet, you'll find that all your input raster
 
 <br>
 
-In the *Results Scenario* you should also find that the **Spatial Multiprocessing** datasheet under the **Data Preparation** tab has been populated, along with the **Site Data** and **Retained Covariate List** datasheets under the **Variable Reduction** tab.  
+In the results scenario you should also find that the **Spatial Multiprocessing** datasheet under the **Data Preparation** tab has been populated, along with the **Site Data** and **Retained Covariate List** datasheets under the **Variable Reduction** tab. 
 
 Since we opted for multiprocessing, we can see that a tiling raster has been created and added to the **Spatial Multiprocessing** datasheet. This tiling raster is used to clip other spatial layers into smaller rectangular blocks, effectively creating more manageable processing sizes. 
 
 **Site Data** is an output of the *Site Data Preparation* stage of the *Pipeline* and provides site specific values for each covariate. The **Retained Covariate List** is an output of the *Variable Reduction* stage of the *Pipeline* and lists the candidate variables that were considered during model fitting.
 
 <br>
 
-### Model Outputs
+### Model outputs
 
-To view spatial outputs, move to the results panel at the bottom left of the **Library Explorer** window. Under the **Maps** tab, double-click on **Map Outputs** to visualize the map.
+To view spatial outputs, move to the results panel at the bottom left of the *Explorer* window. Under the **Maps** tab, double-click on **Map Outputs** to visualize the map.
 
 <img align="middle" style="padding: 3px" width="292" src="assets/images/library-explorer-maps.png">
 
 The first two maps are *Probability* maps showing model-predicted habitat suitability values in geographic space. Values in the legend on the left-hand side of the screen represent probabilities as percentages. The two maps represent outputs using *GLM* and *Random Forest* statistical analyses. One map will be visible for each modeling approach.
 
 <img align="middle" style="padding: 3px" width="939" src="assets/images/map-outputs.png">
 
-Under the **Map Outputs** tab, you will also find the **Model Outputs** tab. Outputs include *Response Curves*, *Standard Residuals*, *Residuals Smooth*, *Calibration*, *ROC/AUC*, *AUCPR*, *Confusion Matrix*, and *Variable Importance*. These outputs provide information on model performance and offer quick comparison of different statistical models. 
```

Comment:
These should now be under the Images tab. Add this part back in, but guide the user to open the Images outputs instead